mattermost.py

```diff
@@ -20,11 +20,23 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 # THE SOFTWARE.
 
+from __future__ import print_function
```

Comment:
No need to use __future__ to support print(). Both python 2 and 3 support print().